go/kbfs/libkbfs/init.go

```diff
@@ -659,7 +663,64 @@ func doInit(
 		return nil, fmt.Errorf("Unexpected mode: %s", params.Mode)
 	}
 
-	initMode := NewInitModeFromType(mode)
+	return NewInitModeFromType(mode), nil
+}
+
+func getCleanBlockCacheCapacity(
+	ctx context.Context, kbCtx Context, params InitParams,
+	log logger.Logger) uint64 {
+	cap := params.CleanBlockCacheCapacity
+
+	// Use the capacity from the config file if none is provided on
+	// the command line.
+	if cap == 0 {
+		config := kbCtx.GetEnv().GetConfig()
+		capInt, ok := config.GetIntAtPath(configBlockCacheMemMaxBytesStr)
+		if ok {
```

Comment:
there's no fallback if it's not in the config file. Intentional?

Reply:
I imagine it might be nice to default to the values we had before.

Reply:
@jakob223 fall back to what?  `cap` is still set to whatever the parameter value was.  (Note that this is setting `capInt`, not `cap`, and it's `cap` that's returned below.

Reply:
Oh, I see. 👍